packages/react-components/src/components/CameraButton.tsx

```diff
@@ -94,13 +71,14 @@ export const CameraButton = (props: CameraButtonProps): JSX.Element => {
   }, [localVideoViewOption, onToggleCamera]);
 
   return (
-    <DefaultButton
+    <ControlBarButton
       {...props}
       disabled={props.disabled || waitForCamera}
       onClick={onToggleCamera ? onToggleClick : props.onClick}
-      styles={componentStyles}
-      onRenderIcon={onRenderIcon ?? defaultRenderIcon}
-      onRenderText={showLabel ? onRenderText ?? defaultRenderText : undefined}
+      onRenderOnIcon={props.onRenderOnIcon ?? onRenderCameraOnIcon}
+      onRenderOffIcon={props.onRenderOnIcon ?? onRenderCameraOffIcon}
```

Comment:
Did you mean to check for `props.onRenderOffIcon`?

Reply:
I did nice catch!

Reply:
updated